core/docs/d7yxc-operator_manual.md

```diff
@@ -102,6 +102,22 @@ To ensure that an SDK will be used with a compatible CLI and runner:
 1. If using a pre-installed CLI, install the CLI at that version as detailed in [Versioning](#versioning).
 1. Once the runner and/or CLI are setup, you are safe to upgrade your SDK to the newest version.
 
+### Can I run the Dagger Engine as a "rootless" container?
+
+No. "Rootless mode" means running the Dagger Engine as a container without the `--privileged` flag. In this case, the container would not have "root" privileges for all devices on the host system. Currently, the Dagger Engine cannot be run as a rootless container; network, platform and volume constraints related to rootless usage would significantly limit its capabilities and performance.
+
+#### Platform constraints
+
+The Dagger Engine uses Buildkit. In order to run it in rootless mode, it is necessary to use [Rootlesskit](https://github.com/rootless-containers/rootlesskit), an experimental tool that aims to replace the "root" user with a "fake root" user using user namespaces. However, depending on the host operating system, some features might not be fully supported.Since Dagger is intended to run consistently everywhere, introducing platform-specific incompatibilities would be contrary to this goal.
```

Comment:
```suggestion
```

I think this section is mostly redundant with the below ones since the other sections talk about the various platform-related problems we run into w/ rootless containers.

If we do prefer to leave this section in, we don't need to mention rootlesskit specifically since there's many ways of approaching it. We can just say "it is necessary to use user namespaces".